act-rules-format.bs

```diff
@@ -106,45 +106,68 @@ An ACT Rule MUST have a description that is in plain language, and provides a br
+
+Accessibility references can be [accessibility requirements](#accessibility-requirements) or [related references](#related-references). Rule authors MAY list any number of accessibility references that the rule has been written to test. Accessibility references are omitted when rules do not directly relate to particular accessibility references but can be used together with other rules as input, or when rules are designed to be part of a [=composite rule=]. This listing can be extended when rules are applied in other contexts.
 
-Accessibility requirements are just that: A requirement that a particular web page conforms to for it to be considered accessible. A common example of accessibility requirements are the WCAG 2.1 success criteria. Some organizations have additional requirements that come from different sources, such as laws, internal standards, etc. These too are considered accessibility requirements which can be tested using the ACT Rules Format.
 
+Accessibility Requirements {#accessibility-requirements}
+--------------------------------------------------------
 
-Accessibility Requirements List {#accessibility-requirements-list}
-------------------------------------------------------------------
+Accessibility requirements are normative requirements for conformance, such as WCAG 2.1 Success Criteria. Some organizations have additional requirements, such as laws and internal standards. These too are considered accessibility requirements.
 
-An ACT Rule SHOULD list an identifier and reference for each accessibility requirement that it maps to.
+An ACT Rule MUST list an identifier and reference for each accessibility requirement that it is written to directly test. That is, when the outcome of a rule determines meeting or not meeting the accessibility requirement it has been written for. This MAY be omitted when rules are designed to be part of a [=composite rule=].
 
 <div class=example>
-  <p><strong>Example</strong>: The outcome of this rule affects conformance to the following WCAG 2.1 success criteria:</p>
+  <p><strong>Example</strong>: The outcome of this rule indicates conformance to the following WCAG 2.1 success criteria:</p>
   <ul>
     <li>[1.3.1 Info and Relationships](https://www.w3.org/TR/WCAG21/#info-and-relationships)</li>
     <li>[4.1.2 Name, Role, Value](https://www.w3.org/TR/WCAG21/#name-role-value)</li>
   </ul>
 </div>
 
-One reason why an [=atomic rule=] might not list accessibility requirements is because it is part of a [=composite rule=], and only a `failed` [=outcome=] in the composite rule would mean not satisfying the accessibility requirement. Another reason might be that an ACT Rule is designed to evaluate a best practice that is not part of any accessibility standard.
 
+Related References {#related-references}
```

Comment:
I don't think "references" is the right term here. I'm having a hard time coming up with an alternative, as I'm not entirely sure what this section is even for.

Reply:
I addressed your other comments on this section. Hopefully it is now clearer.